pkg/services/alerting/engine.go

```diff
@@ -105,6 +107,10 @@ func (e *Engine) processJob(grafanaCtx context.Context, job *Job) error {
 
 	done := make(chan struct{})
 
+	span := opentracing.StartSpan("alerting")
```

Comment:
this can be more descriptive of what the span actually represents, which seems to be the execution of one job.